share/zm-testagent.lsb

```diff
@@ -20,9 +20,9 @@ PIDFILE=${ZM_BACKEND_PIDFILE:-/var/run/zonemaster/zm-testagent.pid}
 USER=${ZM_BACKEND_USER:-zonemaster}
 GROUP=${ZM_BACKEND_GROUP:-zonemaster}
 
-#export ZM_BACKEND_TESTAGENT_LOGLEVEL='warning'  # Set this variable to override the default log level
+#export ZM_BACKEND_TESTAGENT_LOGLEVEL='info'  # Set this variable to override the default log level
 
-testagent_args="--logfile=$LOGFILE --outfile=$OUTFILE --pidfile=$PIDFILE --user=$USER --group=$GROUP"
+testagent_args="--loglevel=$ZM_BACKEND_TESTAGENT_LOGLEVEL --logfile=$LOGFILE --outfile=$OUTFILE --pidfile=$PIDFILE --user=$USER --group=$GROUP"
```

Comment:
If you pass it directly to the arguments, shouldn't the variable be exported then ?

Reply:
Since the variable is only used locally it doesn't need to be exported. And since the default value is defined in zonemaster_backend_testagent I don't want to define it again in the init scripts - for the sake of having a single source of truth.

But you're completely right that there is something fishy going on here! I'm setting --loglevel to the empty string when I ZM_BACKEND_TESTAGENT_LOGLEVEL is undefined. I'll fix that.